builtin/providers/aws/structure.go

```diff
@@ -622,12 +623,19 @@ func flattenOptions(list []*rds.Option) []map[string]interface{} {
 				r["db_security_group_memberships"] = dbs
 			}
 			if i.OptionSettings != nil {
+				spew.Sdump("Options HERE %s", i.OptionSettings)
 				settings := make([]map[string]interface{}, 0, len(i.OptionSettings))
 				for _, j := range i.OptionSettings {
-					settings = append(settings, map[string]interface{}{
-						"name":  *j.Name,
-						"value": *j.Value,
-					})
+					spew.Sdump("Option HERE %s", j)
```

Comment:
looks like some extra debugging things here. In general it's OK to use `spew` in our tests, but we kind of made a decision to not include it in another files (so it won't ship with the Terraform binary as a dependency) .

_Turns out_ spew is in some files, but I'll remove those 😆